website/docs/reference/sources/journald.md.erb

```diff
@@ -34,7 +34,23 @@
 <%- end -%>
 ## How It Works [[sort]]
 
-<%= component_sections(component) %>
+### Checkpointing
+
+Vector checkpoints the journal position after every successful read. This
+ensures that Vector resumes where it left off if restarted, preventing data
+from being read twice. The checkpoint positions are stored in the data
+directory which is specified via the
+[global `data_dir` option][docs.configuration#data-directory] but can be
+overridden via the `data_dir` option in the `journald` source directly.
 
+### Communication with systemd journal
 
+To ensure the `journald` source works across all platforms, Vector interacts
+with the Systemd journal via the `journalctl` utility. This is accomplished by
+spawning a [subprocess][urls.rust_subprocess] that Vector diligently interacts
+with. Taking care to restart the process and handle failures accordigly. If
```

Comment:
We don't actually handle restarting at this point. It has been discussed, but we didn't come up with a satisfactory way of handling initial `journalctl` failures causing rapid restarting and CPU burn.

Reply:
Ok. Can't we retry with a backoff? Should we open an issue for this?

Reply:
Definitely should open an issue for this.